src/main.js

```diff
@@ -21,12 +25,18 @@ import 'vue-material/dist/vue-material.css'
 import 'vue-material/dist/theme/default.css'
 import 'vue-snotify/styles/material.css'
 
+import 'onsenui/css/onsenui.css'
+import 'onsenui/css/onsen-css-components.css'
+
 import store from './store'
 
 Vue.use(VueAxios, axios)
 Vue.use(VueMaterial)
 Vue.use(Snotify)
 
+Vue.use(VueOnsen)
+Vue.component('custom-toolbar', MobileCustomToolbar) // Common toolbar
```

Comment:
same here, we should find a way to avoid loading of not required components when a specific platform has been detected. For example this line could not be useful for Desktop UI.